pkg/sql/schemachanger/scplan/internal/rules/helpers.go

```diff
@@ -405,12 +407,6 @@ func or(predicates ...elementTypePredicate) elementTypePredicate {
 	}
 }
 
-func not(predicate elementTypePredicate) elementTypePredicate {
-	return func(e scpb.Element) bool {
-		return !predicate(e)
-	}
-}
```

Comment:
This might not be useful right now, but it keeps coming up from time to time. Let's leave it here for the time being?

Reply:
It will cause a CI linter failure -- object `rules.not` is not used -- as happened in [this run](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/7157298?showRootCauses=false&expandBuildChangesSection=false&expandBuildProblemsSection=true&expandBuildDeploymentsSection=true&expandBuildTestsSection=true).

Is there easy way, like a directive, to skip?